ui/src/app/workflows/components/workflow-dag/workflow-dag.tsx

```diff
@@ -185,7 +187,11 @@ export class WorkflowDag extends React.Component<WorkflowDagProps, WorkflowDagRe
             return <Loading />;
         }
         const {nodes, edges} = this.prepareGraph();
-        this.layoutGraph(nodes, edges);
```

Comment:
minor: I'd create a new method called `layoutGraph` and switch in that

Reply:
Done